packages/astro/src/core/config/schema.ts

```diff
@@ -311,8 +311,10 @@ export const AstroConfigSchema = z.object({
 						fallback: z.record(z.string(), z.string()).optional(),
 						detectBrowserLanguage: z.boolean().optional().default(false),
 						// TODO: properly add default when the feature goes of experimental
-						fallbackControl: z.enum(['none', 'redirect', 'render']).optional(),
-					})
+                        routingStrategy: z
+                            .enum(['prefix-always', 'prefix-other-locales'])
```

Comment:
This is kinda unrelated to the PR, but what's the difference between `z.enum` and `z.union`? I notice we use both 🤔

Reply:
I am not an expert, I just checked their documentation. With unions you do `z.union([z.string, z.numer])`, and you support multiple types. `z.enum` accepts just strings.